fairlearn/metrics/_metric_frame.py

```diff
@@ -334,17 +340,46 @@ def __init__(
                 raise ValueError(_DUPLICATE_FEATURE_NAME.format(name))
             nameset.add(name)
 
-        # Create the basic results
-        result = DisaggregatedResult.create(
-            data=all_data,
-            annotated_functions=annotated_funcs,
-            sensitive_feature_names=self._sf_names,
-            control_feature_names=self._cf_names,
-        )
-
-        # Build into cache
         self._result_cache = dict()
-        self._populate_results(result)
+        if n_bootstrap_samples is None:
+            assert ci_quantiles is None, "Can't have ci_quantiles not None"
```

Comment:
This message is a bit unclear, maybe mention that they perhaps forgot to set `n_bootstrap_samples`. btw, I believe that in scikit-learn it is more common to simply ignore parameters if the "main" parameter is not set (e.g. ['average' in `roc_auc_score`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score).